src/modm/platform/clock/stm32/rcc.cpp.in

```diff
@@ -290,6 +290,29 @@ modm::platform::Rcc::enablePll(PllSource source, const PllFactors& pllFactors, u
 %%endif
 }
 
+%% if target["family"] == "f4"
+bool
+modm::platform::Rcc::enablePllI2s(PllSource, const PllI2sFactors& pllFactors, uint32_t waitCycles)
```

Comment:
`PllSource` is not used (because it is not configurable?).

Reply:
Good observation! Initially, I thought I will use it to do some sanity check on the frequency of the input clock of the I2S PLL. But for now the user has to check manually that the right input clock is provided. The parameter is removed.